src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -27,7 +27,16 @@ public interface IDelegatableTableValue
         /// <param name="parameters">delegation parameters.</param>
         /// <param name="cancel"></param>
         /// <returns></returns>
+        [Obsolete($"Use  {nameof(ExecuteQueryAsync)} instead.")]
         Task<IReadOnlyCollection<DValue<RecordValue>>> GetRowsAsync(IServiceProvider services, DelegationParameters parameters, CancellationToken cancel);

```

Comment:
>DelegationParameters [](http://example.com/codeflow?start=95&length=20)

should we add count to DelegationParameterFeatures? 
So that EnsureOnlyFeatures catches it?